doc/doxygen/src/advanced-build-system-tricks.md

```diff
@@ -40,6 +40,78 @@ You can configure your own files that will be parsed by the build system main
 * Override default targets
 
 
+Handling multiple boards with udev-rules                {#multiple-boards-udev}
+========================================
+
+When developing and working with multiple boards the default `PORT` configuration
```

Comment:
This description handles multiple different boards not multiple time the same one, which would require another variable to handle different numbers.
It could be precised that it is only for this or add an explanation for multiple time the same board.

Reply:
Also it is one way of doing and not the only way. It could be tweaked in the wording for saying the nuance. It is only my taste, depends on real current users.